gatsby-config.js

```diff
@@ -7,6 +7,8 @@ require("dotenv").config({
   path: `.env.${activeEnv}`,
 })
 
+let siteUrl = `https://adriantirusli.me/blog/`
```

Comment:
Also, keep in mind with this line since `siteMetadata` don't give us any `siteUrl` or equivalent node.

Reply:
Fixed!